src/js/classic-editor.js

```diff
@@ -1,6 +1,6 @@
 /* global wpAjax */
 
-const TermsOrder = {
+const ClassicEditor = {
```

Comment:
@DavidCramer 

There are still many references to jQuery in this script.
Can't we remove them?

Reply:
@pereirinha Oops, I messed some 🤦 - forgot about the `jQuery` over the `$`